README.md

```diff
@@ -16,3 +22,12 @@ dapr run --app-id receiverLog --app-protocol http --app-port 8002 --dapr-http-po
 ```
 node app.js
 ```
+
+### Mini klient + server js (mozliwe ze sie w portach zagubilam ;) )
```

Comment:
If it works then you did not xd, maybe it is wise to leave that digression off the main branch xD